tests/kohana/Log/WriterTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/**
+ * Tests Kohana Logging API - writing to Syslog
+ *
+ * @group kohana
+ * @group kohana.core
+ * @group kohana.core.logging
+ * @group kohana.core.logging.syslog
+ *
+ * @package    Kohana
+ * @category   Tests
+ * @author     Kohana Team
+ * @copyright  (c) 2008-2012 Kohana Team
+ * @license    http://kohanaframework.org/license
+ */
+class Kohana_Log_WriterTest extends Unittest_TestCase
```

Comment:
`Kohana_Log_AbstractWriterTest` is a base class containing filtering tests
https://github.com/kohana/core/commit/8f3cebeb022dfe6682869e33e30b3aeb043b7937